src/main/java/org/folio/persist/AbstractRepository.java

```diff
@@ -39,8 +38,8 @@ public Future<String> save(String id, T entity) {
     return postgresClient.save(tableName, id, entity);
   }
 
-  public void save(String id, T entity, Handler<AsyncResult<T>> replyHandler) {
-    postgresClientFuturized.save(tableName, id, entity, replyHandler);
+  public Future<T> save(String table, String id, T entity) {
```

Comment:
There is a `tableName` field in the class, so we don't need it in the method args.